packages/common/core-interfaces/Removing-IFluidRouter.md

```diff
@@ -117,3 +117,8 @@ const entryPoint = await container.getEntryPoint();
 | `requestResolvedObjectFromContainer`                                                         | 2.0.0-internal.7.0.0 | 2.0.0-internal.8.0.0 |
 | `getDefaultObjectFromContainer`, `getObjectWithIdFromContainer` and `getObjectFromContainer` | 2.0.0-internal.7.0.0 | 2.0.0-internal.8.0.0 |
 <!-- prettier-ignore-end -->
+
+The removal of the following items will need to wait for the LTS version of the `Loader` to reach "2.0.0-internal.7.0.0":
```

Comment:
Can you add another sentence or two explaining why?  These all must lie along or near the loader-runtime boundary somehow?